plugins/catalog/src/components/CreateComponentButton/CreateComponentButton.tsx

```diff
@@ -23,9 +23,7 @@ import { useRouteRef } from '@backstage/core-plugin-api';
 export const CreateComponentButton = () => {
   const createComponentLink = useRouteRef(createComponentRouteRef);
 
-  if (!createComponentLink) return null;
-
-  return (
+  return !createComponentLink ? null : (
```

Comment:
Can we flip this ternary expression and not start with a negation in the precendent? Because I feel other teammates might easily miss the `!` at the beginning.

```suggestion
  return createComponentLink ? (
```

and a `:` somewhere lower.